tests/common/plugins/sanity_check/recover.py

```diff
@@ -23,6 +23,15 @@ def reboot_dut(dut, localhost, cmd):
     else:
         reboot_type = REBOOT_TYPE_COLD
 
+    if reboot_with_running_golden_config:
+        logging.info("Reboot DUT with the running golden config")
+        dut.copy(
+            src="/etc/sonic/running_golden_config.json",
```

Comment:
Does the src file always exist? Do we need to add checker?

Reply:
Agree, suggest to copy golden config with best effort, if no src file, print a warning msg

Reply:
Updated